extensions/websockets-next/server/deployment/src/test/java/io/quarkus/websockets/next/deployment/WebSocketServerProcessorTest.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.websockets.next.deployment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class WebSocketServerProcessorTest {
+
+    @Test
+    public void testGetPath() {
+        assertEquals("/foo/:id", WebSocketServerProcessor.getPath("/foo/{id}"));
+        assertEquals("/foo/:bar-:baz", WebSocketServerProcessor.getPath("/foo/{bar}-{baz}"));
+        assertEquals("/ws/v:version", WebSocketServerProcessor.getPath("/ws/v{version}"));
+        assertEquals("/foo/v:bar/:bazand:alpha_1-:name",
+                WebSocketServerProcessor.getPath("/foo/v{bar}/{baz}and{alpha_1}-{name}"));
```

Comment:
If you convert `{baz}and{alpha}` to `:bazand:alpha`, it is not going to work as expected, right?

Reply:
I mean, if the underlying implementation only supports colon-delimited parameters, it has to define where the parameter ends. I guess it's on a non-alphanumeric character. We should be able to detect a situation when the closing brace `}` is followed by an alphanumeric character and fail in that case.

Reply:
You're right but (1) I don't think we can fix that and (2) path params are usually not used within the same part of the path. The vertx-web docs mention `"/flights/:from-:to"` which would work because _"Parameter names consist of any alphabetic character, numeric character or underscore."_ by default. This is something that needs to be documented too.

Reply:
I'm not saying we should _fix_ it, I'm saying we should detect the situation and fail (at build).

Reply:
https://github.com/quarkusio/quarkus/issues/39182